process_pr.py

```diff
@@ -1109,20 +1149,27 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
     # to use information during the actual comment loop
     for comment in issue.get_comments():
         all_comments.append(comment)
-        if bot_cache or technical_comment:
-            continue
         if comment.user.login.encode("ascii", "ignore").decode() != cmsbuild_user:
             continue
         comment_msg = comment.body.encode("ascii", "ignore").decode() if comment.body else ""
         first_line = "".join([l.strip() for l in comment_msg.split("\n") if l.strip()][0:1])
         if re.match(ISSUE_SEEN_MSG, first_line):
             already_seen = comment
-            bot_cache = read_bot_cache(comment_msg)
-            print("Read bot cache from already seen comment:", comment)
-        elif re.match(CMSBOT_TECHNICAL_MSG, first_line):
-            technical_comment = comment
-            bot_cache = read_bot_cache(comment_msg)
-            print("Read bot cache from technical comment:", comment)
+            bot_cache = extact_bot_cache(comment_msg)
+            if bot_cache:
+                print("Read bot cache from already seen comment:", comment)
+                technical_comments = [comment]
+                break
```

Comment:
@iarspider , need to drop `break` here to make sure we process all comments in this loop